tools/sdk-ai-bots/offline-evaluation.yml

```diff
@@ -62,7 +62,7 @@ extends:
                 displayName: 'check go'
               - task: AzureCLI@2
                 inputs:
-                  azureSubscription: 'azure-chat-bot-test'
+                  azureSubscription: 'azuresdkqabot-dev'
```

Comment:
Maybe we could add `readme.md` to introduce about the pipeline including overview and how to use this pipeline.